Makefile

```diff
@@ -1,6 +1,7 @@
 SDCC ?= sdcc
 STCCODESIZE ?= 4089
-SDCCOPTS ?= --iram-size 256 --code-size $(STCCODESIZE) --xram-size 0 --data-loc 0x30 --disable-warning 126 --disable-warning 59
+SDCCOPTS ?= --iram-size 256 --code-size $(STCCODESIZE) --xram-size 0 --data-loc 0x30 --disable-warning 126 --disable-warning 59 \
+    -DDEBUG -DWITH_ALT_LED9 -DWITHOUT_LEDTABLE_RELOC
```

Comment:
Yes, I would like to add more optional features, so I will figure out how to specify these without editing the Makefile.  Thanks for accepting the PR.